lib/helpers/lib/each-openapi-module.js

```diff
@@ -137,8 +137,10 @@ function child_entries(node) {
                 format_additionalProperties(node.additionalProperties) ]]
             : []
 
-    const via_allOf = node.allOf?.flatMap(
-        a => Object.entries(a.properties || {})) || []
+    // allOf can be nested, so call child_entries recursively
+    const via_allOf = (node.allOf || []).flatMap(subNode =>
+      child_entries(subNode)
```

Comment:
Huh, it actually *was* that easy...
I was very much "can't program my way out of a paper bag" last time I looked at this...
Thanks for taking this on!